meta/send.rb

```diff
@@ -224,6 +224,22 @@
 end
 
 Mutant::Meta::Example.add do
+  source 'foo.values_at(a, b)'
```

Comment:
True. This should probably be `self` too then.

Yes. In general I think that we can significantly improve meta by making it more consistent and incorporating https://github.com/mbj/mutant/issues/415 and https://github.com/mbj/mutant/issues/395